From 1058038534715a884d8eab9b28c060a7b80d9ea1 Mon Sep 17 00:00:00 2001
From: Christopher Degawa <ccom@randomderp.com>
Date: Fri, 9 May 2025 00:00:22 -0500
Subject: [PATCH 2/2] uiddefs: limit AL_NO_UID_DEFS to DEVPKEY stuff

unfortunately, those symbols seem to be the only ones necessary that are
not predefined in a library provided by mingw-w64.

Signed-off-by: Christopher Degawa <ccom@randomderp.com>
---
 core/uiddefs.cpp | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/core/uiddefs.cpp b/core/uiddefs.cpp
index e52a9ae3..c9b8d2f1 100644
--- a/core/uiddefs.cpp
+++ b/core/uiddefs.cpp
@@ -26,4 +26,9 @@ DEFINE_PROPERTYKEY(PKEY_AudioEndpoint_GUID, 0x1da5d803, 0xd492, 0x4edd, 0x8c, 0x
 #endif
 #endif
 
+#else
+
+#include <initguid.h>
+#include <devpkey.h>
+
 #endif /* AL_NO_UID_DEFS */
-- 
2.49.0

